.mailmap

```diff
@@ -1305,6 +1305,7 @@ V1krant <46847915+V1krant@users.noreply.github.com>
 Vaibhav Bhat <vaibhav.bhat2097@gmail.com> VBhat97 <vaibhav.bhat2097@gmail.com>
 Vaishnav Damani <vaishnavdamani3496@gmail.com>
 Valeriia Gladkova <valeriia.gladkova@gmail.com>
+Varenyam Bhardwaj <varenyambhardwaj123@gmail.com>
```

Comment:
Change this to:
```
Varenyam Bhardwaj <varenyambhardwaj123@gmail.com> Varenyam Bhardwaj <65490150+airpods69@users.noreply.github.com>
```

Reply:
You also need to add a line like
```
Varenyam Bhardwaj <varenyambhardwaj123@gmail.com> Airpods <varenyambhardwaj123@gmail.com>
```
because Airpods is the name that you've been using in your commits.

Reply:
Ahh that makes more sense.